sktime/transformations/panel/shapelet_transform.py

```diff
@@ -38,9 +39,35 @@ class Shapelet:
         The calculated information gain of this shapelet
     data: array-like
         The (z-normalised) data of this shapelet.
+    backend : str, optional
```

Comment:
I think this is the wrong class.

This should be in `RandomShapeletTransform`, no?